build/packaging/linux/bin/ksend

```diff
@@ -0,0 +1,254 @@
+        OPTIONS="$OPTIONS --ws"
+        if [ ! -z $WS_API ]; then
+            OPTIONS="$OPTIONS --wsapi $WS_API"
+        fi
+        if [ ! -z $WS_PORT ]; then
+            OPTIONS="$OPTIONS --wsport $WS_PORT"
+        fi
+        if [ ! -z $WS_ADDR ]; then
+            OPTIONS="$OPTIONS --wsaddr $WS_ADDR"
+        fi
+        if [ ! -z $WS_ORIGINS ]; then
+            OPTIONS="$OPTIONS --wsorigins $WS_ORIGINS"
+        fi
+    fi
+
+    # Cypress network => NETWORK_ID is null && NETWORK = "cypress"
+    # Baobab network => NETWORK_ID is null && NETWORK = "baobab"
+    # Else => private network
+    if [[ -z $NETWORK_ID ]]; then
+        if [[ $NETWORK == "baobab" ]]; then
+            OPTIONS="$OPTIONS --baobab"
+        elif [[ $NETWORK == "cypress" ]]; then
+            OPTIONS="$OPTIONS --cypress"
+        else
+            echo
+            echo "[ERROR] network id is not specified and network is not available."
+            echo "Available network: baobab, cypress"
+            exit 1
+        fi
+    else
+        OPTIONS="$OPTIONS --networkid $NETWORK_ID"
+        echo "[INFO] creating a private network: $NETWORK_ID"
+        if [[ ! -z $NETWORK ]]; then
+            echo
+            echo "[WARN] ignoring the specified network: $NETWORK"
+        fi
+    fi
+
+    if [ ! -z $DATA_DIR ]; then
+        OPTIONS="$OPTIONS --datadir $DATA_DIR"
+    fi
+
+    if [ ! -z $PORT ]; then
+        OPTIONS="$OPTIONS --port $PORT"
+    fi
+
+    if [ ! -z $SUBPORT ]; then
+        OPTIONS="$OPTIONS --subport $SUBPORT"
+    fi
+
+    if [ ! -z $SERVER_TYPE ]; then
+        OPTIONS="$OPTIONS --srvtype $SERVER_TYPE"
+    fi
+
+    if [ ! -z $VERBOSITY ]; then
+        OPTIONS="$OPTIONS --verbosity $VERBOSITY"
+    fi
+
+    if [ ! -z $TXPOOL_EXEC_SLOTS_ALL ]; then
+        OPTIONS="$OPTIONS --txpool.exec-slots.all $TXPOOL_EXEC_SLOTS_ALL"
+    fi
+
+    if [ ! -z $TXPOOL_NONEXEC_SLOTS_ALL ]; then
+        OPTIONS="$OPTIONS --txpool.nonexec-slots.all $TXPOOL_NONEXEC_SLOTS_ALL"
+    fi
+
+    if [ ! -z $TXPOOL_EXEC_SLOTS_ACCOUNT ]; then
+        OPTIONS="$OPTIONS --txpool.exec-slots.account $TXPOOL_EXEC_SLOTS_ACCOUNT"
+    fi
+
+    if [ ! -z $TXPOOL_NONEXEC_SLOTS_ACCOUNT ]; then
+        OPTIONS="$OPTIONS --txpool.nonexec-slots.account $TXPOOL_NONEXEC_SLOTS_ACCOUNT"
+    fi
+
+    if [ ! -z $SYNCMODE ]; then
+        OPTIONS="$OPTIONS --syncmode $SYNCMODE"
+    fi
+
+    if [ ! -z $MAXCONNECTIONS ]; then
+        OPTIONS="$OPTIONS --maxconnections $MAXCONNECTIONS"
+    fi
+
+    if [ ! -z $LDBCACHESIZE ]; then
+        OPTIONS="$OPTIONS --db.leveldb.cache-size $LDBCACHESIZE"
+    fi
+
+    if [[ ! -z $SC_MAIN_BRIDGE ]] && [[ $SC_MAIN_BRIDGE -eq 1 ]]; then
+        OPTIONS="$OPTIONS --mainbridge --mainbridgeport $SC_MAIN_BRIDGE_PORT"
+        if [[ ! -z $SC_MAIN_BRIDGE_INDEXING ]] && [[ $SC_MAIN_BRIDGE_INDEXING -eq 1 ]]; then
+            OPTIONS="$OPTIONS --childchainindexing"
+        fi
+    fi
+
```

Comment:
I think KSEN/KSPN also need to SubBridgeOption like below.

```
    if [[ ! -z $SC_SUB_BRIDGE ]] && [[ $SC_SUB_BRIDGE -eq 1 ]]; then
        OPTIONS="$OPTIONS --subbridge --subbridgeport $SC_SUB_BRIDGE_PORT --chaintxperiod $SC_TX_PERIOD --chaintxlimit $SC_TX_LIMIT "
        OPTIONS="$OPTIONS --parentchainid $SC_PARENT_CHAIN_ID"
    fi
```

Reply:
Subbridge support in ksen/kspn will be added in the next PR.